ios/RCTWebRTC/VideoCaptureController.m

```diff
@@ -165,4 +203,19 @@ - (AVCaptureDeviceFormat *)selectFormatForDevice:(AVCaptureDevice *)device
     return selectedFormat;
 }
 
+- (void)throttleFrameRateForDevice:(AVCaptureDevice *)device {
+    NSError *error = nil;
+    
+    [device lockForConfiguration:&error];
+    if (error) {
+        RCTLog(@"[VideoCaptureController] Could not lock device for configuration: %@", error);
+        return;
+    }
+    
+    device.activeVideoMinFrameDuration = CMTimeMake(1, 20);
```

Comment:
Are these in the right order? We are setting this to 15fps min and 20 fps max, right?

Reply:
Never mind: "A device’s minimum frame duration is the reciprocal of its maximum frame rate. "

Reply:
yeah, found this a bit confusing too